client/app/bundles/HelloWorld/store/helloWorldStore.jsx

```diff
@@ -0,0 +1,24 @@
+import { createStore, applyMiddleware } from 'redux';
+import rootReducer from '../reducers/index';
+import Immutable from 'immutable';
+
+// See
+// https://github.com/gaearon/redux-thunk and http://redux.js.org/docs/advanced/AsyncActions.html
+// This is not actually used for this simple example, but you'd probably want to use this
+// once your app has asynchronous actions.
+import thunkMiddleware from 'redux-thunk';
```

Comment:
Please use this linter setup: https://www.npmjs.com/package/eslint-config-shakacode

12.0.0-beta.1 

I just updated it to the latest.